packages/www/pages/app/stream/[id].tsx

```diff
@@ -258,7 +258,10 @@ const ID = () => {
       } else if ("errors" in info) {
         throw new Error(info.errors.toString());
       }
+
+      // @ts-ignore
       setStream(info.stream);
+      // @ts-ignore
       setStreamOwner(info.user);
```

Comment:
What's this about? 👀

Reply:
just testing the vercel deployment, will revert it back this one :)